.github/workflows/review.yml

```diff
@@ -32,6 +32,8 @@ jobs:
           DOCKER_BUILD_ARG_NEXT_PUBLIC_API_BASE_URL: 'https://palvelutarjotin-api.test.kuva.hel.ninja/graphql'
           DOCKER_BUILD_ARG_NEXT_PUBLIC_UNIFIED_SEARCH_BASE_URL: 'https://unified-search.test.kuva.hel.ninja/search'
           DOCKER_BUILD_ARG_NEXT_PUBLIC_CMS_BASE_URL: 'https://kultus.content.api.hel.fi/graphql'
+          DOCKER_BUILD_ARG_NEWSLETTER_BASE_URL: 'https://api.createsend.com/api/v3.2'
+          DOCKER_BUILD_ARG_NEWSLETTER_APIKEY: ${{ secrets.GH_QA_NEWSLETTER_APIKEY }}
```

Comment:
Should these be in env-section?

Reply:
They are not public after the implementation was moved to next js api.

Reply:
I think this is the correct place